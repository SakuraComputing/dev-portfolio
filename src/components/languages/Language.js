import React from 'react';

const Language = (props) => {
    return (
        <div>
            <li className="item">
                <span className="title"><strong>{props.language.title}: </strong></span>
                <span className="level">{props.language.level} <br className="visible-xs"/>
                    {
                        'â˜…'.repeat(props.language.rating)
                    }
                </span>
            </li>

        </div>
    );
};

export default Language;
