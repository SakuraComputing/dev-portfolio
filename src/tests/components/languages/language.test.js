import React from 'react';
import { shallow } from 'enzyme';
import Language from '../../../components/languages/Language';

describe('Language Section', () => {
    it('should render correctly', () => {
        const language = jest.fn();
        const wrapper = shallow(<Language language={language}/>);
        expect(wrapper).toMatchSnapshot();
    });
});