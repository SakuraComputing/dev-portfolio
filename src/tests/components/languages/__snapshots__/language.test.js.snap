// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Language Section should render correctly 1`] = `
<div>
  <li
    className="item"
  >
    <span
      className="title"
    >
      <strong />
    </span>
    <span
      className="level"
    >
       
      <br
        className="visible-xs"
      />
      <i
        className="fa fa-star"
      />
       
      <i
        className="fa fa-star"
      />
       
      <i
        className="fa fa-star"
      />
       
      <i
        className="fa fa-star"
      />
       
      <i
        className="fa fa-star"
      />
       
    </span>
  </li>
</div>
`;
